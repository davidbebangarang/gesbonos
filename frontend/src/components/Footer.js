import React, {Component} from 'react'

class Footer extends Component {

    render(){
        return(   
            <div style={{ bottom: "0",width:"100%"}}>                          
                <footer className="page-footer font-small blue mt-5 panel-footer foot"> 
                    <div className="footer-copyright text-center py-3">Â© 2020 Copyright:
                        <a className="link" href="..."> gesbonos.com</a>
                    </div>
                </footer>
            </div>       
        )
    }
}
export default Footer;