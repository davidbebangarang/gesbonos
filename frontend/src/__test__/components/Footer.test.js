import React from 'react';
import {mount} from 'enzyme';
import { create }from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('<Footer />',()=>{
    const footer = mount(<Footer />);
    test('render del componente Footer',()=>{
      
        expect(footer.length).toEqual(1);
    });

    test ('render del titulo',()=>{
        expect(footer.find('.link').text()).toEqual('Gesbonos.com');
    })
});

describe('Footer Snapshot', () => {
    test('Comprobar la UI del componente Footer', () => {
      const footer = create(<Footer />);
      expect(footer.toJSON()).toMatchSnapshot();
    });
  });
