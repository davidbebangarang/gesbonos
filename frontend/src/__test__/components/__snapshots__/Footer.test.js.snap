// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Comprobar la UI del componente Footer 1`] = `
<div
  style={
    Object {
      "bottom": "0",
      "width": "100%",
    }
  }
>
  <footer
    className="page-footer font-small blue mt-5 panel-footer foot"
  >
    <div
      className="footer-copyright text-center py-3"
    >
      Â© 2020 Copyright:
      <a
        className="link"
        href="..."
      >
        Gesbonos.com
      </a>
    </div>
  </footer>
</div>
`;
